packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -385,7 +385,7 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
       "es.object.create",
       "es.object.freeze",
     ]),
-    stringify: define("json/stringify", ["es.json.stringify"], "es.symbol"),
+    stringify: define("json/stringify", ["es.json.stringify"]),
```

Comment:
It's not a bug or something like that. In early versions of `core-js@3`, `JSON.stringify` logic was only in `es.symbol` module to fix `JSON.stringify` with symbols. After that, many other fixes of  `JSON.stringify` logic, [well-formed unicode](https://github.com/tc39/proposal-well-formed-stringify), [`JSON.rawJSON`](https://github.com/tc39/proposal-json-parse-with-source) support, etc. were added and the main part of `JSON.stringify` polyfill was moved to `es.json.stringify`.

For modern `core-js` versions, `es.symbol` is not required, however, it could break compatibility with early `core-js` versions.